apps/nextjs/src/app/(dashboard)/dashboard/layout.tsx

```
@@ -1,69 +1,17 @@
-      <SidebarRail />
-    </Sidebar>
-  );
-}
-
-function DashboardContent({ children }: Readonly<ChildrenProps>) {
-  return (
-    <SidebarInset className="border-l-[1px] bg-transparent">
-      <DashboardHeader />
-      <Suspense fallback={<DashboardContentSkeleton />}>
+    <section
+      role="status"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)